src/renderers/webgl/WebGLRenderLists.js

```diff
@@ -174,24 +174,26 @@ function WebGLRenderLists( properties ) {
 
 	let lists = new WeakMap();
 
-	function get( scene, camera ) {
+	function get( scene, renderCallDepth ) {
 
-		const cameras = lists.get( scene );
 		let list;
 
-		if ( cameras === undefined ) {
+		if ( lists.has( scene ) === false ) {
 
 			list = new WebGLRenderList( properties );
-			lists.set( scene, new WeakMap() );
-			lists.get( scene ).set( camera, list );
+			lists.set( scene, [] );
+			lists.get( scene ).push( list );
```

Comment:
I think we can just do this?

```js
lists.set( scene, [ list ] );
```

Reply:
I'll change it locally.

Reply:
The same change can be applied in `WebGLRenderStates`, too.

Reply:
Sorry, you were faster than my comment 😅  af07e8ad159a7106da5d2984813e3a282a8e4b75